packages/x-date-pickers/src/internals/components/PickerProvider.tsx

```diff
@@ -116,6 +123,25 @@ export interface PickerContextValue<
    * If the picker does not have a field (if it is a static picker) or is not open, the view should be focused.
    */
   autoFocus: boolean;
+  /**
+   * The views that the picker has to render.
+   * It is equal to the picker `views` prop—if defined.
+   * Otherwise, a default set of views is provided based on the component you are using:
+   * - Date Pickers: ['year', 'day']
+   * - Time Pickers: ['hours', 'minutes']
+   * - Date Time Pickers: ['year', 'day', 'hours', 'minutes']
+   * - Date Range Pickers: ['day']
+   * - Date Time Range Pickers: ['day', 'hours', 'minutes']
+   */
+  views: readonly TView[];
+  /**
+   * The currently rendered view.
+   */
+  view: TView | null;
+  /**
+   * The view showed when first opening the picker.
```

Comment:
```suggestion
   * The first view shown when opening the picker.
```

Reply:
The meaning wouldn't be the same
This is the view that is rendered, when you open the picker for the first time AFAIK.

Reply:
Fair point, it could be understood differently. 👌 
We can disregard this comment. 👍

Reply:
Otherwise:

```suggestion
   * The first view shown when opening the picker for the first time.
```

Reply:
Sounds great. 👍